tests/hcaptcha.spec.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ReactTestUtils from 'react-dom/test-utils';
+import HCaptcha from '../src/index';
+import {getMockedHcaptcha, MOCK_EKEY, MOCK_TOKEN, MOCK_WIDGET_ID} from './hcaptcha.mock';
+
+
+const TEST_PROPS = {
+    sitekey: '10000000-ffff-ffff-ffff-000000000001',
+    theme: 'light',
+    size: 'invisible',
+    tabindex: 0,
+};
+
+describe('hCaptcha', () => {
```

Comment:
These look good, we also want to verify that react props get set, rendered, and called correctly.

Reply:
`el renders after api loads and a widget id is set` asserts that render is called appropriately with the passed properties.